rejistry++/src/NKRecord.cpp

```diff
@@ -58,6 +58,8 @@ namespace Rejistry {
         int32_t offset = (int32_t)getDWord(CLASSNAME_OFFSET_OFFSET);
         uint16_t length = getWord(CLASSNAME_LENGTH_OFFSET);
 
+		// Not sure why we are performing this check. I haven't found any documentation on
```

Comment:
Please replace tabs with spaces here and in the rest of the PR

Reply:
Updated in https://github.com/sleuthkit/sleuthkit/pull/1569/commits/3cbc5c2ba0a658fff60a63f90929c3fed162766a